### ISSUE TITLE
Problematic openshift-it profile

### ISSUE THREAD

**Issue by bartoszmajsak** [openshiftio/booster-common#18]
Maybe someone can help me understand what it stands for in all of the boosters, but at least for [wfswarm-rest-http](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http/blob/18aea0e717058934385691cd48aec89eac37df02/pom.xml#L185-L208) it just sets env variable for the tests. 

I would like to propose to either make it enabled by default or not rely on env variable for the tests (just hardcode the app name). Thoughts?

**Comment by Ladicek** [comment]
Hardcoding doesn't work, because Fabric8 Launcher lets you select your own name. There needs to be a system property.

Now that doesn't mean there needs to be an `openshift-it` profile. I believe the `openshift-it` profile _only_ exists because of OSIO having issues when the tests are included in the `openshift` profile (which, I believe, is what everyone would like to have).

**Comment by bartoszmajsak** [comment]
I doubt "OSIO issues" were ever closely investigated though.... https://github.com/fabric8io/fabric8-pipeline-library/pull/381

TL;DR;

- f8-pipeline-library had a bug
- making `openshift-it` profile available by default 

all tests are passing.

**Comment by Ladicek** [comment]
> I doubt "OSIO issues" were ever closely investigated though

That's most probably true.

I think everyone would be happy if we could merge the `openshift` and `openshift-it` profiles into one (called `openshift`), but of course I didn't ask :-)

**Comment by bartoszmajsak** [comment]
If there are no objections I can do that. Just need a list of all the repos :)

**Comment by Ladicek** [comment]
I'm not the one to decide on that, should probably start a discussion internally. Oh well, discussions...

**Comment by cescoffier** [comment]
The profiles has been merged before. We had to re-split them because of OSIO.

**Comment by bartoszmajsak** [comment]
Thanks for sharing this fact @cescoffier. I wasn't aware of that. Do you have a link to an issue at hand so that I could understand the reasoning from OSIO perspective?

**Comment by bartoszmajsak** [comment]
I think with the latest changes on f8-pipeline-library in this PR https://github.com/fabric8io/fabric8-pipeline-library/pull/381 we are fine living with two profiles.  But still, would be worth to understand the actual reason for the split
